test/test_nn.py

```diff
@@ -1452,6 +1452,26 @@ def test_spectral_norm(self):
         self.assertTrue(hasattr(m, 'weight'))
         self.assertTrue('weight' in m._parameters)
 
+    def test_spectral_norm_eval(self):
```

Comment:
Could you also test somewhere that 
1. output doesn't change after adding and immediately removing SN
2. after adding SN, fwd a couple iteration, output doesn't change after removing SN.